services/data/src/links/RestAPILink/queryToRequestOptions.ts

```diff
@@ -33,7 +46,10 @@ export const queryToRequestOptions = (
     return {
         method: getMethod(type),
         body: requestBodyForContentType(contentType, query),
-        headers: requestHeadersForContentType(contentType),
+        headers: {
+            ...requestHeadersForContentType(contentType),
+            Accept: getAcceptHeader(query),
+        },
```

Comment:
Perhaps cleaner to call the `getAcceptHeader` helper in `requestHeadersForContentType`

Reply:
🤔 On second thought.... I can see why you kept this separate, since the helper is suffixed with `ForContentType` and the accept header does not actually depend on the content type. Still it would be in good company... `requestBodyForContentType` has committed the same sin.

On balance I don't think one option is significantly better than the other. And hopefully we can remove this helper soon anyway. So just leave it if you'd like.